### DISCUSSION TITLE
Amasty_Stockstatus

### DISCUSSION THREAD

**Discussion by jorgb90** [breezefront/community#182] [3. Integrations]
We already use the Amasty_Stockstatus extension for I believe 10 years and never found a better way to manage stock statuses then through this extension. Its working with Breeze for just simple products out of the box, but for configurable products its not. Please add support for this extension.

**Comment by dimdimich** [comment]
Thank you for this request. Please contact Amasty also and check if they are willing to help. We will check information in your ticket from our side.
